decidim-elections/app/views/decidim/elections/elections/show.html.erb

```diff
@@ -49,6 +49,12 @@ edit_link(
   </div>
 </div>
 
+<% if onboarding %>
+  <%= render partial: "decidim/elections/votes/onboarding_modal" %>
+<% end %>
+
 <%= cell("decidim/elections/election_preview", election) %>
 
 <%= cell("decidim/elections/election_results", election) %>
+
+<%= javascript_include_tag("decidim/elections/onboarding") %>
```

Comment:
💭 I'd move this `js` include to the partial `onboarding_modal` as the script is to open it, and so where the partial is rendered the script is also included 😄

Reply:
Nice!!!!!